zio-http/src/main/scala/zhttp/service/DecodeJResponse.scala

```diff
@@ -13,7 +14,9 @@ trait DecodeJResponse {
   def decodeJResponse(jRes: JFullHttpResponse): Either[Throwable, UHttpResponse] = Try {
     val status  = Status.fromJHttpResponseStatus(jRes.status())
     val headers = Header.parse(jRes.headers())
-    val content = HttpContent.Complete(jRes.content().toString(HTTP_CHARSET))
+    val bytes   = new Array[Byte](jRes.content().readableBytes)
+    jRes.content().readBytes(bytes)
+    val content = HttpContent.Complete(Chunk.fromArray(bytes))
```

Comment:
```suggestion
    val content = HttpContent.complete(jRes.content())
```
Add a helper function to create `Complete` which can copy the data from a byteBuf to a Chunk.